### ISSUE TITLE
UpdateNotification module contains unlocalizable word "commits"

### ISSUE THREAD

**Issue by balassy** [MichMich/MagicMirror#1368]
The `modules/default/updatenotification/updatenotification.js` file contains the following lines:
```
var subtextHtml = this.translate("UPDATE_INFO", {
  COMMIT_COUNT: this.status.behind + " " + ((this.status.behind == 1) ? "commit" : "commits"),
  // ….
});
```

Even though the localization files contain an entry for `UPDATE_INFO`, the word `commit` or `commits` always appears in the displayed message.

I recommend removing the words, and substituting only the number into the `COMMIT_COUNT` placeholder. Note however that this may be a breaking change for some existing localization message.

**MagicMirror Version**: 2.4.1

**Expected Results**: Non-English words never appear in localized messages.

**Actual Results**: The word `commit` or `commits` always appear in the update notification message.

Thank you for creating and maintaining this project open source!

**Comment by fewieden** [comment]
What is your suggestion to replace it with, as commit is the technical term for it?

**Comment by balassy** [comment]
Yes, "commit" is the technical term for it, however in some languages that word may change its form when used in plural or in this particular sentence. I can also imagine that a translator can find another word for this term that is less technical for an average user. 

My recommendation is to leave it to the languages and the localization files how they want to use this word and substitute only the `this.status.behind` value into the `COMMIT_COUNT` placeholder.

**Comment by MichMich** [comment]
Feel free to send a non-breaking PR for this.

**Comment by balassy** [comment]
I've just submitted PR #1383 to change this behavior.

**Comment by rejas** [comment]
Since PR #1383 got merged this I think this issue can be closed, @MichMich @balassy ? :-)

**Comment by balassy** [comment]
Yes, this can be closed, PR #1383 fixed the issue. Thank you.
